gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixTask.java

```diff
@@ -213,9 +213,17 @@ private Integer getPartitionForHelixTask(TaskDriver taskDriver) {
 
   @Override
   public void cancel() {
-    log.info("Gobblin helix task cancellation invoked.");
+    log.info("Gobblin helix task cancellation invoked for jobId {}.", jobId);
     if (this.task != null ) {
-      this.task.cancel();
+      try {
+        this.task.cancel();
+        log.info("Gobblin helix task cancellation completed for jobId {}.", jobId);
+      } catch (Throwable t) {
+        log.info("Gobblin helix task cancellation for jobId {} failed with exception.", jobId, t);
```

Comment:
As @autumnust has pointed out to me, the pattern of catching an exception to log and re-throw the same exception is an anti-pattern. I would recommend reading this SO post: https://softwareengineering.stackexchange.com/questions/365427/try-catch-log-rethrow-is-anti-pattern
In your case, I think you are catching the exception to log the jobId. In this case, per the SO post, it is recommended to wrap the original exception into another exception with the additional context and propagate it up the call stack.